build_helpers/publish_docker_multi.sh

```diff
@@ -64,6 +65,7 @@ docker build --cache-from freqtrade:${TAG_FREQAI} --build-arg sourceimage=${CACH
 docker tag freqtrade:$TAG_PLOT ${CACHE_IMAGE}:$TAG_PLOT
 docker tag freqtrade:$TAG_FREQAI ${CACHE_IMAGE}:$TAG_FREQAI
 docker tag freqtrade:$TAG_FREQAI_RL ${CACHE_IMAGE}:$TAG_FREQAI_RL
+docker tag freqtrade:$TAG_FREQAI_RL ${CACHE_IMAGE}:$TAG_FREQAI_TORCH
```

Comment:
@xmatthias please have a look at this to ensure I am complying with your wishes. The goal here is not to push a new image, but to simply add another tag to the RL image.

Reply:
well i don't mind this necessarily (though it's pointless) - but it'll have no impact, as it's creating a new tag on the cache image (the one nobody knows about, and more importantly, nobody should be using).

Reply:
Docker part is fine now - and should create a new tag.